packages/svelte/tests/parser-legacy/samples/action-duplicate/output.json

```diff
@@ -31,4 +31,4 @@
 			}
 		]
 	}
-}
+}
```

Comment:
can we undo all these changes so the PR is less noisy?

Reply:
Sorry, fixed.